File: src/dailyprogrammer/easy/_391/abacaba.js

function sequencer(n) {
    //given number of iterations, returns final sequence
    seq = "";
    for (let i = 1; i <= n; i++) {
        middleAscii = 'a'.charCodeAt(0) + i - 1;
        middle = String.fromCharCode(middleAscii);
        seq = seq + middle + seq;
    }
    return seq;
}

function test() {
    console.log('a'.charCodeAt(0) + 1);
    console.log(String.fromCharCode(98));
}

console.log(sequencer(5));
test();

File: src/dailyprogrammer/easy/_391/abacaba.java
package dailyprogrammer.easy._391;

public class abacaba {
    public static void main(String[] args) {
        System.out.println(sequencer(5));
    }
    public static String sequencer(int n) {
        //given number of sequences, returns final sequence
        String seq = "";
        int i = 1;
        while(i <= n)
            seq = seq + (char)((int)'a' + i++ - 1) + seq;
        return seq;
    }
}
